diff --git a/types/transaction.d.ts b/types/transaction.d.ts
index 0cb533f021125654c284d24148c479e2aebd495d..32225b40c0a5ccdfbdb8e20213764d77ae95650d 100644
--- a/types/transaction.d.ts
+++ b/types/transaction.d.ts
@@ -57,7 +57,7 @@ export interface Transaction extends TransactionContext, Omit<Span, 'setName' |
      * Was this transaction chosen to be sent as part of the sample?
      * @deprecated Use `spanIsSampled(transaction)` instead.
      */
-    sampled?: boolean;
+    sampled?: boolean | undefined;
     /**
      * @inheritDoc
      */
diff --git a/types-ts3.8/transaction.d.ts b/types-ts3.8/transaction.d.ts
index af8fc6dd85e05cfa13072ca4df4a81857de3743a..aa0babea4a495f582b6aba2376d5e0dd1acbbbee 100644
--- a/types-ts3.8/transaction.d.ts
+++ b/types-ts3.8/transaction.d.ts
@@ -57,7 +57,7 @@ export interface Transaction extends TransactionContext, Pick<Span, Exclude<keyo
      * Was this transaction chosen to be sent as part of the sample?
      * @deprecated Use `spanIsSampled(transaction)` instead.
      */
-    sampled?: boolean;
+    sampled?: boolean | undefined;
     /**
      * @inheritDoc
      */
