diff --git a/index.d.ts b/index.d.ts
index 756a025eb97d7d2819e3688bb16f5118432f7921..eeaa95025d08dc5b44f4e865e4b89006637ddc5c 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,13 +1,4 @@
-import type {
-  ClassicConfig,
-  FlatConfig,
-} from '@typescript-eslint/utils/ts-eslint';
+import { ESLint } from 'eslint';
 
-import type rules from './rules';
-
-declare const cjsExport: {
-  configs: Record<string, ClassicConfig.Config>;
-  meta: FlatConfig.PluginMeta;
-  rules: typeof rules;
-};
+declare const cjsExport: ESLint.Plugin;
 export = cjsExport;
