diff --git a/lib/findvs.js b/lib/findvs.js
index e7619598670af48bec8b8a8cce6d39afefb9a209..93754cf9dccfedc1a80d79ea653d7e2ab8448cfe 100644
--- a/lib/findvs.js
+++ b/lib/findvs.js
@@ -126,7 +126,8 @@ function findModernVS() {
     'v1.0',
     'powershell.exe'
   );
-  const cs = path.resolve(__dirname, '..', 'deps', 'Find-VisualStudio.cs');
+  const cs = process.env.FIND_VISUALSTUDIO_PATH ??
+    path.resolve(__dirname, '..', 'deps', 'Find-VisualStudio.cs');
   const args = [
     '-ExecutionPolicy',
     'Unrestricted',
